src/bench/addrman.cpp

```diff
@@ -118,8 +104,8 @@ static void AddrManSelectFromAlmostEmpty(benchmark::Bench& bench)
     AddrMan addrman{EMPTY_NETGROUPMAN, /*deterministic=*/false, ADDRMAN_CONSISTENCY_CHECK_RATIO};
 
     // Add one address to the new table
-    CService addr = ResolveService("250.3.1.1", 8333);
-    addrman.Add({CAddress(addr, NODE_NONE)}, ResolveService("250.3.1.1", 8333));
+    CService addr = Lookup("250.3.1.1", 8333, false).value();
+    addrman.Add({CAddress(addr, NODE_NONE)}, Lookup("250.3.1.1", 8333, false).value());
```

Comment:
Any reason why not just reuse the already looked up service?
```suggestion
    const CService service{Lookup("250.3.1.1", 8333, false).value()};
    addrman.Add({CAddress(service, NODE_NONE)}, service);
```

Reply:
No, going to change it